### DISCUSSION TITLE
Design System CMS

### DISCUSSION THREAD

**Discussion by timford22** [cloudscape-design/components#73] [General]
Awesome job with the design system documentation. What CMS did you use to create it?

**Comment by gethinwebster** [comment]
Thanks for your question, and we’re glad you like the documentation! The site is built using Cloudscape components, and the content is pulled together from a variety of different sources, including static JSX, a headless CMS, and auto-generated content from the Cloudscape component TypeScript definitions.

**Reply by timford22** [reply]
Thanks for your response! What I am looking for is what CMS tool you used for the documentation? Did you use zeroheight, invision or something else?

**Reply by gethinwebster** [reply]
We don't use any design system-focused documentation system like zeroheight or InVision. Most of our component documentation is stored in source code (for example, https://github.com/cloudscape-design/components/blob/main/src/alert/interfaces.ts), as is a lot of the visual foundation such as design tokens (https://github.com/cloudscape-design/components/blob/main/style-dictionary/visual-refresh/index.ts). In general, we find this solution of having documentation and implementation as close to each other as possible helps a lot with maintainability. Some additional documentation content is stored in a headless CMS (Contentful). This is all pulled together into the documentation website as a purpose-built React application.
